api/v1/common.go

```diff
@@ -16,11 +16,11 @@
 
 package v1
 
-type DriverPhase string
+type ProductPhase string
```

Comment:
is it for [this](https://github.com/IBM/ibm-block-csi-operator/pull/256#discussion_r896975155)?

Reply:
yes

Reply:
I think we can stay with `Driver`

Reply:
done